src/mlfmu/fmu_build/templates/onnx_fmu/onnxFmu.hpp

```diff
@@ -55,7 +55,14 @@ class OnnxFmu : public cppfmu::SlaveInstance
     Ort::Env env;
     Ort::RunOptions run_options;
     Ort::Session session_ {nullptr};
+
+    // std::wstring onnxPath_;
+    // store path as wstring for Windows or as char * for Linux
+#ifdef _WIN32
     std::wstring onnxPath_;
+#else
+    std::string onnxPath_;
+#endif
```

Comment:
```suggestion

    // store path as wstring for Windows or as char * for Linux
#ifdef _WIN32
    std::wstring onnxPath_;
#else
    std::string onnxPath_;
#endif
```

Reply:
Looks like you forgot to remove the commented out `std::wstring onnxPath_`

Reply:
Fixed.